src/alfasim_score/converter/alfacase/_tests/test_convert_casing.py

```diff
@@ -0,0 +1,93 @@
+from typing import Any
+
+import json
+import pytest
+from pathlib import Path
+from pytest_regressions.data_regression import DataRegressionFixture
+
+from alfasim_score.converter.alfacase.convert_alfacase import ScoreAlfacaseConverter
+from alfasim_score.converter.alfacase.score_input_reader import ScoreInputReader
+
+
+def test_convert_casing_list(
+    data_regression: DataRegressionFixture,
+    score_input_example: ScoreInputReader,
+) -> None:
+    builder = ScoreAlfacaseConverter(score_input_example)
+    casings = builder._convert_casing_list()
+    data_regression.check(
```

Comment:
What about creating a method that prepares these objects for regression, like [this](https://github.com/ESSS/all4well/blob/master/alfasim_integration/source/python/alfasim_integration/integration/test_utils.py#L27)?

Reply:
I thought about it, do you think I should do this here, or another task just to do this ?

Reply:
Better do it now. We weill probably need it later

Reply:
Very well, working on it.

Reply:
done